ample/ensembler/single_model.py

```diff
@@ -153,8 +150,8 @@ def _generate_residue_scorelist(residue_key, score_key, scores):
         
         :returns: zipped list of residue index plus score
         """
-        assert residue_key in scores[0], "Cannot find residue key in scoresfile"
-        assert score_key in scores[0], "Cannot find score key in scoresfile"
+        assert residue_key in scores[0], "Cannot find residue key \'{}\' in scoresfile header: {}".format(residue_key, scores[0])
```

Comment:
no need to escape the ticks, this is inside double-quotes

Reply:
Good spot